tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
While I agree that a clear public / private API separation is a good thing, I don't think it should come at a cost of using closures, especially when you have to define all reusable variables in that scope, contributing to the current mess. It's also harder to debug using dev tools.

There's a way around this that I've seen in many other open source JS projects — prefixing private methods and members with an `_` (underscore). Since language clearly doesn't support that, I think it's a good enough way of organisational separation. It makes it clear when somebody is calling a private method from outside of it's class and can be easily corrected during the review of the PR.

To make it even more explicit these can be wrapped in a `private` object, making calls look like `this.private.myPrivateMethod()` and `this.private.myPrivateVariable`. Although it might be an overkill and prefixing with `_` is enough.